lib/next/units/validateStatusCode.js

```diff
@@ -10,13 +10,23 @@ const APIARY_STATUS_CODE_TYPE = 'text/vnd.apiary.status-code';
 function validateStatusCode(real, expected) {
   const validator = new TextDiff(real.statusCode, expected.statusCode);
   const rawData = validator.validate();
+  const results = validator.evaluateOutputToResults();
 
   return {
     validator: 'TextDiff',
     realType: APIARY_STATUS_CODE_TYPE,
     expectedType: APIARY_STATUS_CODE_TYPE,
     rawData,
-    results: validator.evaluateOutputToResults()
+    results: results.map((result) =>
```

Comment:
It's safe to map since the output of `evaluateOutputToResults` is `[]` when no errors occurred.